### PR TITLE
New Sub-Spec: B18 Minigunner

### PR DESCRIPTION
**Pull request by Surrealaser**
🆑 by Surrealistik
add: Adds the minigun sub-specialist kit for B18. It's B18 with a damn minigun. Deadly at close to med range, after which it experiences substantial accuracy falloff. There are plans to give it full auto via holding down one's mouse input. Can equip mag harness, rail light, gyro stabilizer and bipod.
tweak: Improved the grenade launcher by slightly increasing its firing speed due to complaints about its shittiness, and to make it more competitive.
/🆑

### CODE REVIEW

File: code/modules/projectiles/updated_projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -935,4 +935,75 @@
...
+	w_class = 5
+	force = 20
+	wield_delay = 15
+	gun_skill_category = GUN_SKILL_SPEC
+	aim_slowdown = SLOWDOWN_ADS_RIFLE
+	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_BURST_ON|GUN_WIELDED_FIRING_ONLY
+	attachable_allowed = list(
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/magnetic_harness,
+						/obj/item/attachable/gyro,
```
Comment (Borisvanmemes) [inline code/modules/projectiles/updated_projectiles/guns/specialist.dm]: Gyros fit on it, kek.

Comment (Surrealaser) [reply code/modules/projectiles/updated_projectiles/guns/specialist.dm]: It's a stabilizer.

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -235,6 +235,16 @@ WEAPONS
 	containername = "\improper Sniper Specialist crate"
 	group = "Weapons"
 
+/datum/supply_packs/specminigun
+	name = "MIC-A7 Vindicator Minigun crate (MIC-A7 x1, Minigun Drums 3x)"
```
Comment (Ghommie) [inline code/datums/supplypacks.dm]: `name = "MIC-A7 Vindicator Minigun crate"`, it's just the gun.

File: code/datums/supplypacks.dm [hunk 3]
```diff
@@ -2019,6 +2029,18 @@ AMMO
 	containername = "ammo crate"
 	group = "Ammo"
 
+/datum/supply_packs/ammo_minigun
+	name = "Vindicator Minigun Ammo Drums (x3)"
+	contains = list(
+					/obj/item/ammo_magazine/minigun,
+					/obj/item/ammo_magazine/minigun,
+					/obj/item/ammo_magazine/minigun)
+	cost = RO_PRICE_CHEAP
```
Comment (Ghommie) [inline code/datums/supplypacks.dm]: Should be `RO_PRICE_KINDA_PRICY`, considering it's one thousand and half ammunitions.

**Pull request merged**